common-test-lib/src/main/java/com/google/cloud/tools/intellij/testing/DelayedSubmitExecutorServiceProxy.java

```diff
@@ -68,8 +68,11 @@ public boolean awaitTermination(long timeout, @NotNull TimeUnit unit)
   }
 
   @Override
-  public void execute(@NotNull Runnable command) {
-    executor.execute(command);
+  public void execute(@NotNull Runnable task) {
+    if (this.task != null) {
```

Comment:
can you explain the changes here? do the changes you made to use a ListeningExecutorService require this?

Reply:
After the change GcsBucketContentEditorPanelTest started to fail - looks like MoreExecutors.listeningDecorator() calls execute() in that test so this essentially disabled effect of delayed executor. I made execute() to delay submission as well and it passes now.